src/ParseServer.js

```diff
@@ -143,6 +143,11 @@ class ParseServer {
     enableSingleSchemaCache = false,
     __indexBuildCompletionCallbackForTests = () => {},
   }) {
+    // verify parse-server is running on node >= 4.6
+    if (process.versions.node < '4.6') {
```

Comment:
Ahhh.... that is a point. Could do an str.split(".") to get each part of the version and compare them one by one?